content/shmem_atomic_add.tex

```diff
@@ -35,11 +35,10 @@
     \apiargument{IN}{ctx}{A context handle specifying the context on which to perform the operation.
         When this argument is not provided, the operation is performed on
         the default context.}
-    \apiargument{OUT}{dest}{The remotely accessible integer data object to be
-        updated  on the remote \ac{PE}.  When using \CorCpp, the type of
-        \dest{} should match that implied in the SYNOPSIS section.}
-    \apiargument{IN}{value}{The value to be atomically added to \dest. When using \CorCpp, the type of \VAR{value} should match that  implied  in
-        the SYNOPSIS  section.}
+    \apiargument{OUT}{dest}{Symmetric address of the destination data object.
+        The type of \dest{} should match that implied in the SYNOPSIS section.}
+    \apiargument{IN}{value}{The value to be atomically added to \dest.
```

Comment:
`dest` is an address, so I'm not sure we're not "adding" `value` to that address.
Perhaps:
```suggestion
    \apiargument{IN}{value}{The value to be atomically added to the destination object.
```

Reply:
Nice catch! I cribbed text from the new bitwise AMOs -- "The operand to the atomic add operation." Let me know if you prefer something different.

Reply:
Sounds pretty good.  If I were to nitpick, I might say that an add operation has 2 operands, so which one is this one?  :wink:  It's good though, I think it's pretty clear this is the sender's operand.